### ISSUE TITLE
fix: Accessibility issue with ColumnOptions, screen reader always announcing name of the filter button of column header as "Filter this column".

### ISSUE THREAD

**Issue by v-malvankars** [microsoft/fast-blazor#3733]
# 🐛 Bug Report

I have used ColumnOptions within TemplateColumn, to show filter and sort button. But when I do the accessibility check with help of screen reader, on filter button focus, it's announcing name of the filter button as "Filter this column".

## 💻 Repro or Code Sample

Below is the fluent button which has this aria-label value as "Filter this column". 
 ```
<fluent-button class="col-options-button stealth" type="button" appearance="stealth" aria-label="Filter this column" b-x1200685t0="" _bl_7e6affd1-6b13-419f-9afc-91a85a01c603="" current-value=""><!--!--><svg style="width: 20px; fill: var(--accent-fill-rest);" focusable="false" viewBox="0 0 20 20" aria-hidden="true"><!--!--><path d="M7.5 13h5a.5.5 0 0 1 .09 1H7.5a.5.5 0 0 1-.09-1h5.09-5Zm-2-4h9a.5.5 0 0 1 .09 1H5.5a.5.5 0 0 1-.09-1h9.09-9Zm-2-4h13a.5.5 0 0 1 .09 1H3.5a.5.5 0 0 1-.09-1H16.5h-13Z"></path></svg></fluent-button>
```
## 🤔 Expected Behavior

There should be a way to overwrite this text to load it denamically based on column header.

## 😯 Current Behavior

Accessibility check with help of screen reader, on filter button focus, it's announcing name of the filter button as "Filter this column".

## 🔦 Context

I am trying to overwrite this aria-label text "Filter this column" to load it denamically based on column header.

## 🌍 Your Environment

OS & Device: [Windows]
Browser [Microsoft Edge]
.NET and Fluent UI Blazor library Version [4.11.8, .NET 8]

**Comment by vnbaaij** [comment]
If a11y is a concern, you should not use the separate filter button but use the `HeaderCellAsButtonWithMenu` option instead.
When using that parameter, you can control the strings with the various `Column...Labels` parameters. Also, the `Tooltip` parameter on the columnscan be used.

**Comment by v-malvankars** [comment]
> If a11y is a concern, you should not use the separate filter button but use the `HeaderCellAsButtonWithMenu` option instead. When using that parameter, you can control the strings with the various `Column...Labels` parameters. Also, the `Tooltip` parameter on the columnscan be used.

As suggested above, I implemented `HeaderCellAsButtonWithMenu` option instead of separate filter button. I refered example from [https://www.fluentui-blazor.net/datagrid-typical](url), but not able to figure out how to navigate through keyboard focus to access Sort/Filter menu items. I tried using arrow keys with/without Alt/Ctrl keys and many other options, but not able to get the way. Can you please help me here?
![Image](https://github.com/user-attachments/assets/1a478314-ef65-46d6-bee4-b42f890f28fa)

**Comment by philliphoff** [comment]
@v-malvankars I see the same thing with v4.11.9.
